tracetool/Modules/RTLog.py

```diff
@@ -0,0 +1,48 @@
+import os
+from util import *
+
+
+class traceHandler:
```

Comment:
For the files you placed in 'modules', can you explain what each file is doing? I am especially confused by RTLog.py and surface.py, but I would also like to understand relationship of ftrace.py with the rest since I am guessing libva.py should use ftrace.py.

Reply:
the files in modules are dynamic mod for trace tool.
from functional side, these modules could be spited into 2 type:
1. trace input file reader/parser, e.g ftrace.py for linux ftrace format, there are more input file format support in embargo.
2. trace event handler, e.g. libva.py, media.py, these are python to process events.
these module did not link to each other, the core.py connect them together.
such module design is more easy to add new module for specific trace process. 

RTLog.py is for process media runtime log trace event, rtlog auto generate manifest, so more easy to use. 
e.g. https://github.com/intel-innersource/drivers.gpu.unified/blob/790f3317d9dbcdefbbe200dfaeaae251fd1da258/Source/media/media_softlet/agnostic/common/vp/hal/bufferMgr/vp_allocator.cpp#L824
surface.py is for tracking surface id/attributes(like height/width/format) and surface usage in workload.
these are trace event handlers to process specific trace events

Reply:
> surface.py
Why it's under modules? It looks to be an util, not a module....

> RTLog.py is for process media runtime log trace event

I still don't get what's that. Actually, can you, please, make a table and add it to `readme.md`. Something like:

| Module | Notes |
| -------- | ------ |
| i915.py | Parses i915 logs.  See instructions below on how to enable. |
| ftrace.py | Parses what? |
| libva.py | Parses libva logs. See instructions below to enable |
| media.py |  Parses iHD media-driver logs. See instructions below to enable. |
| RTlog.py | Parses what? |

For each of the modules it should be clear: 1) which output it parses, 2) how to enable this output if it's not a default?

Also, are RTLog.py and media.py both parses something specific to iHD driver? If so, I would suggest to rename these since there are other venders and other drivers. `ihd.py` and `ihd-rtlog,py` would make more sense. Though I need to understand what is rtlog to recommend further.

Reply:
ok, will update